src/parser/postgresparser.cpp

```diff
@@ -288,36 +288,23 @@ parser::GroupByDescription* PostgresParser::GroupByTransform(List* group,
   result->columns = new std::vector<expression::AbstractExpression*>();
   for (auto cell = group->head; cell != nullptr; cell = cell->next) {
     Node* temp = reinterpret_cast<Node*>(cell->data.ptr_value);
-    switch (temp->type) {
-      case T_ColumnRef:
```

Comment:
By the way, postgres documentation says that any expression is valid in group by column as long as it is formed from the input columns. I think only ParamExpressions is invalid in this case. See https://www.postgresql.org/docs/8.0/static/sql-select.html. I am also surprised that weird queries like `select a>1 from a group by a>1` also work in postgres. I guess the rewriter has some rules to eliminate group by.